src/app/util/ember-compatibility-functions.cpp

```diff
@@ -121,11 +122,14 @@ EmberAfAttributeType BaseType(EmberAfAttributeType type)
                       "chip::DataVersion is expected to be uint32_t, change this when necessary");
         return ZCL_INT32U_ATTRIBUTE_TYPE;
 
-    case ZCL_EVENT_NO_ATTRIBUTE_TYPE:  // Event Number
-    case ZCL_FABRIC_ID_ATTRIBUTE_TYPE: // Fabric Id
-    case ZCL_NODE_ID_ATTRIBUTE_TYPE:   // Node Id
-    case ZCL_BITMAP64_ATTRIBUTE_TYPE:  // 64-bit bitmap
-    case ZCL_EPOCH_US_ATTRIBUTE_TYPE:  // Epoch Microseconds
+    case ZCL_EVENT_NO_ATTRIBUTE_TYPE:   // Event Number
+    case ZCL_FABRIC_ID_ATTRIBUTE_TYPE:  // Fabric Id
+    case ZCL_NODE_ID_ATTRIBUTE_TYPE:    // Node Id
+    case ZCL_BITMAP64_ATTRIBUTE_TYPE:   // 64-bit bitmap
+    case ZCL_EPOCH_US_ATTRIBUTE_TYPE:   // Epoch Microseconds
+    case ZCL_POSIX_MS_ATTRIBUTE_TYPE:   // POSIX Milliseconds
+    case ZCL_SYSTIME_MS_ATTRIBUTE_TYPE: // System time Milliseconds
+    case ZCL_SYSTIME_US_ATTRIBUTE_TYPE: // System time Microseconds
         static_assert(std::is_same<chip::EventNumber, uint64_t>::value,
```

Comment:
Do the new types need static assertions here too?

Reply:
No, because they are not mapped to any interesting C++ type.  They're just mapped to `uint64_t`.  If that changes, then yes,